src/components/Shared/DocHeader.tsx

```diff
@@ -0,0 +1,77 @@
+import { cx } from 'class-variance-authority'
+import { H1 } from '@/components/MDX/H1'
+import { P } from '@/components/MDX/P'
+import { Icon } from '@spark-ui/icon'
+import { Tag } from '@spark-ui/tag'
+
+import { Npm } from '@/icons/Npm'
+import { GitHub } from '@/icons/GitHub'
+import { Bug } from '@/icons/Bug'
+import { WarningFill } from '@spark-ui/icons'
+
+export const DocHeader = ({
```

Comment:
There is a `Docs` folder (IMO better to place it there) since this will only used for docs pages